spec/url_validator_spec.rb

```diff
@@ -41,18 +63,123 @@ class Post
     ]
 
     valid_urls.each do |url|
-      it "accept valid url: #{url}" do
+      it 'accept valid url: #{url}' do
         post.url = url
         expect(post).to be_valid
         expect(post.errors.count).to eq(0)
       end
     end
 
     invalid_urls.each do |url|
-      it "reject invalid url: #{url}" do
+      it 'reject invalid url: #{url}' do
```

Comment:
Yes, I replaced all the double quotes from the specs and didn't realize of those exceptions.